vllm/v1/worker/gpu_model_runner.py

```diff
@@ -2169,6 +2169,7 @@ def capture_model(self) -> None:
                 "set -O %s and ensure `use_cudagraph` was not manually set to "
                 "False", CompilationLevel.PIECEWISE)
             return
+        logger.info("Capturing CUDA graphs... This may take 10-60 seconds.")
```

Comment:
the time can vary a lot. i think logging what vllm is doing should be enough.

Reply:
this is highly dependent on the bs. But the default value in vLLM worst case is around 60s